components/scripts/gradle/03-validate-local-build-caching-different-locations.sh

```diff
@@ -47,6 +55,12 @@ main() {
   exit_with_return_code
 }
 
+verify_scans_support_tools() {
+  if [ ! -f "$SCANS_SUPPORT_TOOLS_JAR" ]; then
+    die "ERROR: scans-support-tools.jar is required when running with --disable-build-scan-publishing."
```

Comment:
Adjust jar file name as mentioned in the other comment.

Reply:
Please change to `ERROR: build-scan-support-tool.jar is required when using --disable-build-scan-publishing.`

Reply:
Changed in 959de12707c38c55194c27955284d8a8bc5830ca